### ISSUE TITLE
Crash when opening endnote

### ISSUE THREAD

**Issue by tastytea** [koreader/koreader#9179]
* KOReader version: 2022.05.1
* Device: PocketBook Touch HD 3 (PB632)

#### Issue

Every time i click on a link to an endnote, KOReader crashes. It worked with 2022.03.1 with the same book.

I have configured notes to show in a popup. When i configure them to be on the same page, they show but when i tap anywhere on the page, KOReader crashes (gestures still work).

##### `crash.log` (if applicable)

These are the last lines in the log:

```
06/06/22-18:25:15 INFO  Loading plugins from directory: plugins
ffi.load: sqlite3
06/06/22-18:25:15 WARN  Terminal: can not open /dev/ptmx: Permission denied
06/06/22-18:25:15 WARN  Error when loading plugins/terminal.koplugin/main.lua
06/06/22-18:25:20 INFO  GestureDetector:probeClockSource: Touch event timestamps appear to use CLOCK_REALTIME
Segmentation fault
!!!!
Uh oh, something went awry... (Crash n°1: 06/06/22 @ 18:25:23)
Attempting to restart KOReader . . .
!!!!
```

[crash.log](https://github.com/koreader/koreader/files/8845995/crash.log)

**Comment by tastytea** [comment]
Ah, it is already fixed in 2022.05.1-28. Oopsie!

**Comment by poire-z** [comment]
Thanks for checking with current nightly :)
Probably similar to https://github.com/koreader/crengine/issues/481 - fixed yesterday.
